plugins/example_plugin/plugin.py

```diff
@@ -8,101 +8,109 @@
 import pandas as pd
 import sys
 
-logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, format='\'%(name)s:%(lineno)s\' | %(message)s')
-    
+logging.basicConfig(
+    stream=sys.stdout, level=logging.DEBUG, format="'%(name)s:%(lineno)s' | %(message)s"
+)
 logger = logging.getLogger(os.path.basename(__file__))
 
 
-
-class Plugin(Evaluator):
-
-    """
-    A class used to represent an Animal
-
-    ...
+class Example_Plugin(Evaluator):
```

Comment:
Class need to be called Plugin(Evaluator)

Reply:
done